hknweb/events/templates/events/show_details.html

```diff
@@ -81,7 +83,7 @@ <h3 class="rsvp-list-title">RSVPs</h3>
         <p class="rsvp-list"> Count: {{ rsvps.count }} </p>
       {% endif %}
       {% for rsvp in rsvps %}
-        <p class="rsvp-list">{{ rsvp.user.first_name }} {{ rsvp.user.last_name }} ({{ rsvp.user }})</p>
+        <p class="rsvp-list">{{ rsvp.user.first_name }} {{ rsvp.user.last_name }} ({{ rsvp.user.username }})</p>
       {% endfor %}
     {% elif not rsvpd %}
       <p class="rsvp-list">No rsvps.</p>
```

Comment:
For consistency, maybe display the RSVP count and limit even when there are no RSVPs?